waspc/src/Wasp/Node/Version.hs

```diff
@@ -51,35 +72,33 @@ getNodeVersion = do
     Left procErr ->
       Left
         ( unlines
-            [ procErr,
-              waspNodeRequirementMessage
+            [ "Running `node --version` failed.",
+              indent 2 procErr,
+              "Make sure you have node installed and in your PATH."
             ]
         )
     Right (ExitFailure code, _, stderr) ->
       Left
         ( unlines
             [ "Running `node --version` failed (exit code " ++ show code ++ "):",
-              indent 2 stderr,
-              waspNodeRequirementMessage
+              indent 2 stderr
             ]
         )
-    Right (ExitSuccess, stdout, _) -> case parseNodeVersion stdout of
-      Nothing ->
-        Left
-          ( "Wasp failed to parse node version."
-              ++ " This is most likely a bug in Wasp, please file an issue."
+    Right (ExitSuccess, stdout, _) ->
+      parseNodeVersionOutput stdout
+        & left
+          ( \e ->
+              "Wasp failed to parse node version provided by `node --version`.\n"
+                <> (show e <> "\n")
+                <> "This is most likely a bug in Wasp, please file an issue."
```

Comment:
We have other places where we should also add it then, it feels like bit much for this PR, I would leave it as it is for now. This will anyway almost certainly never happen, and if it does, we will have it happen to us also.